packages/scss/demo/components/tables.html

```diff
@@ -267,15 +267,21 @@ <h2>Clickable lines</h2>
 <em><b>Tip:</b> You can use a <code class="code">&lt;div&gt;</code> & <code class="code">&lt;a&gt;</code> structure if you want to have links on your rows.</em>
 </section>
 
-<!-- Lines actions -->
+<!-- Sortable -->
 <section class="contentSection">
 	<h2>Sortable table</h2>
 	<table class="table">
 		<thead class="table-head">
 			<tr class="table-head-row">
-				<th class="table-head-row-cell mod-sortable is-sorted-descending">Sortable column</th>
-				<th class="table-head-row-cell mod-sortable is-sorted-ascending">Sortable column</th>
-				<th class="table-head-row-cell mod-sortable">Sortable column</th>
+				<th class="table-head-row-cell" aria-sort="descending">
+					<button class="mod-sortable">Sortable column</button>
```

Comment:
There should never be a "mod-xxx" without another class. You should rename either rename this "table-head-row-cell-sortableButton" or have a state class "is-sortable" on the table-head-row-cell that would style the button. (or both actually :) )